app/views/agents/agent_views/user_location_agent/_show.html.erb

```diff
@@ -27,6 +59,13 @@
   </p>
 <% end %>
 
+<div class="hidden checkbox">
+    <label>
+        <input id="toggle" type="checkbox" value="">
```

Comment:
Be good to give this a specific class and select it instead of `.checkbox`.  Maybe `.toggle-accuracy`?